docs/section-7/drivetrain-characterization.md

```diff
@@ -0,0 +1,83 @@
+## Introduction
+This section covers a control method known as Drivetrain Characterization, as described in [this](https://drive.google.com/drive/folders/14UI1WPtFT9KvKjAHt90VEv66ovQtFDvc) paper. In this section, we will develop the Drivetrain Characterization formula along with various tests to determine the constants needed to make the formula work.
+
+## Developing the Drivetrain Characterization Formula
+Our previous method for driving the robot used the joystick values as a percentage of the maximum voltage we want to supply to the motors. But, let's say we wanted a more accurate control method - one based on math and physics. Drivetrain Characterization allows us to more accurately determine the voltage we should supply given a desired velocity and acceleration. This section will require an understanding of the concepts described in [Part 1](https://docs.google.com/presentation/d/14PnR4rYJ5rdAwwJBectMS4YYf6jHjcI0Y-uhp9_ridM/edit#slide=id.g253f61fe15_3_0) and [Part 2](https://docs.google.com/presentation/d/1bMCfHxcyumRmsPNsWSuRBEDfwZfm4lxp9J-wn77V3Ao/edit) of our basic electronic theory presentation.
+
+Brushed DC motors work by running current through a coil of copper wires (known as the windings of the motor) as shown in the figure below. When current passes through these windings, a temporary magnetic field is created. Since there are permanent magnets in the motor, the armature (the part in the center connected to the coil of wires that is allowed to rotate) begins to accelerate towards the magnets (the north end of the armature is attracted to the positive end and the south end is attracted to the negative end). As well, when the armature's acceleration becomes zero, the flow of current reverses and the north end of the armature rotates towards the positive end and the south end rotates towards the negative end. This process repeats as long as there is voltage across the motor.
+
+![DC Motor](https://upload.wikimedia.org/wikipedia/commons/thumb/e/e6/Electric_motor_cycle_1.png/200px-Electric_motor_cycle_1.png)
+
+![Spinning DC Motor](https://upload.wikimedia.org/wikipedia/commons/thumb/8/89/Electric_motor.gif/200px-Electric_motor.gif)
+
+But what prevents the motor from accelerating indefinitely? Well, just as current running through the motor causes the armature to spin, spinning the armature generates current. So, while the motor is spinning, a force, known as the counter electromotive force (or back-EMF), restricts the flow of current. Eventually, back-EMF will cancel out the torque generated by the windings and the rotational speed of the motor will reach a constant value.
```

Comment:
"causes the armature to spin, spinning the armature generates current." -> "causes a torque which tries to accelerate the armature, spinning the armature generates a voltage known as the counter electromotive force (or back-EMF) because it opposes the voltage that was applied to the motor."

Remove "So ,while...restricts the flow of current" 

"cancel out the torque generated by the windings" -> "cancel out the applied voltage. As a result no current will flow through the windings"

Reply:
Would no current be flowing through the windings? I thought that when back-EMF = applied voltage the current through the windings is equal to the free current. It's a small value, but it's definitely not zero.

Reply:
When back-EMF = applied voltage, the current through the windings = 0. No net voltage => no current. 

The free current is the current required to create enough torque to overcome the motor's friction when it is spinning at it's free speed without anything else slowing it down. That free current flows because the back-emf is slightly less than the applied voltage. The corrections I suggested are only correct when the motor is ideal (specifically frictionless, so free current = 0) and does not have any other load (i.e. counter torque that it is working against).

More generally, the "steady state" for the motor is for it to spin at a speed that produces a back-emf which results in a net voltage that produces enough current to create a torque equal to whatever load torque is on the motor. In the case of a motor where the only load torque is it's own friction, the state state speed when the rated voltage (e.g. 12V) is defined to be the free speed and the resulting current is defined to be free current.